storage/applicationinsights/src/main/java/zipkin/storage/applicationinsights/ApplicationInsightsSpanConsumer.java

```diff
@@ -68,6 +77,16 @@ public void accept(List<Span> spans) {
         }
         String res = gson.toJson(jsonElement);
         String msg = "{ \"Span\":" + res + "}";
+        //data model changes
+        telemetry.getContext().getOperation().setId(traceId);
+        telemetry.getContext().getOperation().setName(span.name);
```

Comment:
`span.name` should got to `dependency.name` or `request.name`. Setting it on operation context is optional